elip-0100.mediawiki

```diff
@@ -45,11 +43,11 @@ The new global proprietary types specific to Hardware Wallet Extensions are as f
 ! Versions Allowing Inclusion
 |-
 | Asset Metadata
-| <tt>PSBT_ELEMENTS_HWW_GLOBAL_ASSET_METADATA = 0x0100</tt>
+| <tt>PSBT_ELEMENTS_HWW_GLOBAL_ASSET_METADATA = 0x00</tt>
 | <tt><32 byte asset tag></tt>
 | A 32 byte asset tag which corresponds to this metadata.
 | <tt><compact size uint contractLen><contract><32-byte prevoutTxid><32-bit little endian uint prevoutIndex></tt>
-| The asset metadata including contract and issuance prevout. The parameter <tt>contract</tt> is a JSON document, canonicalized to have its keys sorted lexographically. It is preceded by <tt>contractLen</tt> containing size of the contract in bytes as a compact size unsigned integer. Issuance prevout has two parts: a 32-byte <tt>prevoutTxid</tt> (transaction hash) and a 32-bit little endian unsigned integer <tt>prevoutIndex</tt> (zero-based index of the transaction output).
+| A compact size unsigned integer representing size of the contract in bytes, <tt>contractLen</tt>, is followed with the <tt>contract</tt> itself as a JSON document, canonicalized to have its keys sorted lexicographically. The input of an asset issuance transaction is represented as <tt>prevoutTxid</tt> - a 32-byte previous transaction hash, and <tt>prevoutIndex</tt> - a 32-bit little endian unsigned integer containing a zero-based index of the previous transaction's output.
```

Comment:
> 32-byte previous transaction hash
Please be explicit about which transaction hash and output this is (presumably the issuance tx).

Reply:
<tt>The **input of an asset issuance transaction** is represented as prevoutTxid - [...] previous transaction hash, and prevoutIndex - [...] index of the previous transaction's output.</tt>

It seems to me that the description is quite clear about these parameters being the input's parameters of an issuance transaction. At the same time, these parameters represent the output of another transaction, the _previous_ one with respect to the issuance transaction. In other words, it's an ex-UTXO the issuance transaction spends. That's why these parameters got their name containing 'prevout' ("previous transaction's output"). This naming is common within Elements codebase.

If you think it makes sense to describe it more fully, maybe it would be better to write a paragraph below to not bloat the table with a large piece of text?

Reply:
Ah I didn't read that clearly, confused by the change markup. Thanks I think that's clear enough.